org.alloytools.alloy.core/build.gradle

```diff
@@ -15,12 +15,13 @@ task cup(type: JavaExec ) {
     outputs.files("src/main/java/edu/mit/csail/sdg/parser/CompParser.java", "src/main/java/edu/mit/csail/sdg/parser/CompSym.java")
 
     classpath file( '../cnf/jars/java-cup-11a.jar' )
+    classpath file( '../cnf/jars/java-cup-runtime-0.11-a-czt01-cdh.jar' )
```

Comment:
with the new syntax extension the parser generated by java cup exceeds the maximum method size; with this lib the main parsing method is split into several